osi_referenceline.proto

```diff
@@ -87,31 +87,33 @@ message ReferenceLine
     // ## Adding T coordinates
     //
     // To describe points that are not directly on the polyline, a T coordinate
-    // is added. T is the signed 2D distance (i.e. <code>hypot(A.X-B.X,
-    // A.Y-B.Y)</code>, if A and B are the two points) between the point to
-    // describe and the nearest point on the polyline (this point might either
-    // be on a line segment or at an edge between two line segments). The
-    // distance is positive if the point is left of the polyline (in definition
-    // direction), negative if it is right of it.
-    // The S position of such a point outside the reference line is the same as
-    // the S value of the nearest point on the polyline.
-    //
-    // Notes:
-    // - The "nearest point on the polyline" is determined in 3D (even if the
-    //   resulting T value is only the 2D distance), in order to choose the
-    //   correct point for 3D curves (think reference lines for roads in parking
-    //   decks).
-    // - If there are several "nearest points", the one with the smallest S
-    //   coordinate on the polyline is chosen.
+    // is added. T is the signed 2D distance between the point to describe (P)
+    // and a projected point (P_proj) on the polyline. The T axis (projecting
+    // axis) is the line going through P and the intersection point (I). I is
```

Comment:
What happens if there is no intersection point?

Reply:
Short internal discussion showed, that orthogonal projection isn't 100% correct. To wrap it up:
- There is no intersection point I, if two ReferenceLinePoint neighbors have the same t_axis_yaw
- This t_axis_yaw has to be used as projection direction

We'll update the description accordingly.